### PR TITLE
Add example apps to docs

### PR DESCRIPTION
**Pull request by trishalim**
Looks very boring for now. Will add screenshots when the example apps are looking better. For now, at least people will know these exist.

image

### CODE REVIEW

File: homepage/homepage/app/docs/resources/examples/page.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            slug: "todo",
+        },
+        {
+            name: "Password manager",
+            slug: "password-manager",
+        },
+    ];
+    return (
+        <>
+            <h1>Examples</h1>
```
Comment (aeplay) [inline homepage/homepage/app/docs/resources/examples/page.tsx]:
```suggestion
            <h1>Example Apps</h1>
```

File: homepage/homepage/components/docs/nav.tsx [hunk 2]
```diff
@@ -144,6 +140,17 @@ export function DocNav({ className }: { className?: string }) {
                 </ul>
             </div>
 
+            <div>
+                <DocNavHeader>Resources</DocNavHeader>
+                <ul>
+                    <li>
+                        <DocNavLink href="/docs/resources/examples">
+                            Examples
```
Comment (aeplay) [inline homepage/homepage/components/docs/nav.tsx]:
```suggestion
                            Example Apps
```

**Review by aeplay — APPROVED** [review]
Perfect, thank you!

I think "Example Apps" makes it a bit more noticeable.

If you like that, feel free to accept my suggestions & hit that merge button!

**Comment by trishalim** [comment]
@aeplay That's all good now but I have no permissions to merge

**Comment by aeplay** [comment]
oops, can you merge now? Changed a permission setting.

**Pull request merged**